### PR TITLE
WEB-7856: Bumped lodash version to 4.5.1 to match revolver.

### PR DESCRIPTION
**Pull request by jamescogley**
#### What are the links to relevant tickets?

https://hxshortbreaks.atlassian.net/browse/WEB-7856
#### What does this PR do?

Updates the lodash to version 4.13.1
#### What functional/unit tests does this PR have?

n/a
#### Is there anything a developer should be aware of when reviewing this?

You should be able to pull down the branch and running npm install/update will update the version of lodash to 4.13.1
#### Screenshots (if appropriate)

---
#### Quality checklist (for PR author to complete BEFORE code review):
- [x] I've checked there is appropriate unit test coverage.
- [x] I've updated documentation with any changes to procedures.
- [x] I've checked this work against the requirements of the Jira.
- [x] This change passes all necessary tests (cross browser, automated or otherwise).
- [x] I am ready for this to be code reviewed, merged and tested.
#### Reviewer 1 @hxpaul
- [x] I agree with the assumptions made in the quality checklist.
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked for coding anti-patterns.
- [x] I've checked this work against the requirements of the Jira.
- [x] I don’t believe this work introduces unnecessary technical debt.
- [x] +1 from me!
#### Reviewer 2 @kevinhodges
- [x] I agree with the assumptions made in the quality checklist.
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked for coding anti-patterns.
- [x] I've checked this work against the requirements of the Jira.
- [x] I don’t believe this work introduces unnecessary technical debt.
- [x] +1 from me!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "lib": "./lib"
   },
   "dependencies": {
-    "lodash": "2.4.0"
+    "lodash": "4.5.1"
```
Comment (kevinhodges) [inline package.json]: is there a reason we can't go to 4.13.1?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,2 +1,4 @@
+## **0.2.1**
```
Comment (philmeehan) [inline CHANGELOG.md]: Do we still need this file? We stripped them out of the other repos, or are we keeping them as this is OSS? @kevinhodges

**Comment by hxpaul** [comment]
All the lodash methods that are in this repo still work in the new version so 👍

File: package.json (hunk 1 above)
Comment (jamescogley) [inline package.json]: None that I can see - I was just matching the version used by Revolver.  Bumping to 4.13.1 should be fine.

**Pull request merged**